data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/metrics/OpenSearchSourcePluginMetrics.java

```diff
@@ -18,6 +18,8 @@ public class OpenSearchSourcePluginMetrics {
     static final String PROCESSING_ERRORS = "processingErrors";
     static final String BYTES_RECEIVED = "bytesReceived";
     static final String BYTES_PROCESSED = "bytesProcessed";
+    static final String BASIC_CREDENTIALS_CHANGED = "basicCredentialsChanged";
```

Comment:
Let's give this a more generic name so that it is not tied directly to HTTP Basic authentication. There may be other credential forms that require a chagne. The name `credentialsChanged` seems appropriate.